tutorials/gradleTutorial.md

```diff
@@ -164,6 +164,6 @@ Now that you have a general idea of how to accomplish basic tasks with Gradle, h
  
 * [Official Gradle Documentation](https://docs.gradle.org/current/userguide/userguide.html)
 
-----------------------------------------------------------------------------------------
+---
```

Comment:
I prefer to use the longer line because it appears as a horizontal line even in source code (WYSIWYG)

Reply:
Let's change it to exactly 80 since that's a sensible default for *nix based terminals.